### PR TITLE
Implmented role logic

### PR DESCRIPTION
**Pull request by demosdemosthenous3**

### CODE REVIEW

**Comment by alexyiangou** [comment]
Good job! :+1: Very nice implementation! One comment, run `elsint` like we talked before to clean up all the syntax warnings, and see the individual comments on some lines that I commented :slightly_smiling_face:

File: app/views/login-page/login-page.js [hunk 1]
```diff
@@ -26,33 +28,33 @@ function login(){
...
             
         })
         .then(function(){
-            //FIXME
-            //frameModule.topmost().navigate("views/"+viewToShow+"/"+viewToShow);         
+        
+            if(user.role == constants.customerRole){
+                 viewToShow = constants.customerView;
+            }else{
+                 viewToshow = constants.customerView;
```
Comment (alexandrosyiangou) [inline app/views/login-page/login-page.js]: adminView maybe?

File: app/views/login-page/login-page.js [hunk 2]
```diff
@@ -26,33 +28,33 @@ function login(){
...
+            Toast.makeText("Your account was successfully created").show();
         })
         .catch(function(error){
-            dialogs.alert({
-                message:"Unfortunately we were unable to create your account.",
-                okButtonText:"OK"
-            });
+            //TODO print the actual error
+            //undefined when using error.description or error._bodyTextDescription!!! 
+           Toast.makeText("We were unable to create your account").show();
```
Comment (alexandrosyiangou) [inline app/views/login-page/login-page.js]: What you could also do for these 2 cases is you can try exporting the handleErrors function from the view model and use that here if it accepts the error param

**Review by alexandrosyiangou — CHANGES_REQUESTED** [review]
Good job! :+1: Very nice implementation! One comment, run `elsint` like we talked before to clean up all the syntax warnings, and see the individual comments on some lines that I commented :slightly_smiling_face:

**Review by alexandrosyiangou — APPROVED** [review]

**Pull request merged**